modules/rostests/apitests/kernel32/Pipes.c

```diff
@@ -0,0 +1,129 @@
+/*
+ * PROJECT:         ReactOS api tests
+ * LICENSE:         GNU GPLv2 only as published by the Free Software Foundation
+ * PURPOSE:         Test for pipe (CORE-17376)
+ * PROGRAMMER:      Simone Mario Lombardo
+ */
+
+#include "precomp.h"
+
+static const char* g_PipeName = "\\\\.\\pipe\\rostest_pipe";
+
+#define MINBUFFERSIZE 1
+#define MAXBUFFERSIZE 255
+#define TEST_MESSAGE "Test"
+#define TEST_MESSAGE_SIZE 4
+
+static DWORD g_dwReadBufferSize;
+
+void CheckError(BOOL Success, DWORD dwLastError, const char* errorMsg)
```

Comment:
This function is just bloat: an `ok()` can do all this.

Reply:
Is the result the same here?

Reply:
Which result?

Reply:
Of the function. Will it show the issue the same way?

Reply:
`CheckError(Success, dwLastError, "Something failed\n");`
is the same as
`ok(Success, "Something failed, last error = 0x%lx\n", dwLastError);`
except that `CheckError` outputs that in 2 lines, which is confusing.